sql/plan/join.go

```diff
@@ -51,148 +68,175 @@ type JoinNode interface {
 }
 
 // joinStruct contains all the common data fields and implements the commom sql.Node getters for all join types.
-type joinStruct struct {
+type joinBase struct {
 	BinaryNode
-	Cond       sql.Expression
+	Filter     sql.Expression
```

Comment:
Prefer cond, is there a particular reason you changed this?

Reply:
filters are more mobile in the new join ordering algo, seems easier to just call them all filters. At some point i'd like to split these into a list of filters rather than an `expression.JoinAnd(filters...)`, probably memoized, index selection doesn't have to be as painful as it is right now.